### PR TITLE
Migrate to AndroidX

### PR DESCRIPTION
**Pull request by rpdome**
Migrated using AndroidStudio tool.
5 test cases failed, but it was already failing pre-migration. I opened an issue #701 for tracking.

### CODE REVIEW

File: msal/build.gradle [hunk 1]
```diff
@@ -105,8 +105,8 @@ task sourcesJar(type: Jar) {
 }
 
 dependencies {
-    implementation "com.android.support:appcompat-v7:$rootProject.ext.supportLibraryVersion"
-    implementation "com.android.support:customtabs:$rootProject.ext.supportLibraryVersion"
+    implementation "androidx.appcompat:appcompat:1.0.0"
```
Comment (iambmelt) [inline msal/build.gradle]: If we're going to define this version here, I'd like to see the constant defined in [`versions.gradle`](https://github.com/AzureAD/microsoft-authentication-library-for-android/pull/702/files#diff-25205fe6f2e0d330abcbc45df6b96797R14) either removed or updated and dereferenced here.

File: msal/build.gradle [hunk 2]
```diff
@@ -105,8 +105,8 @@ task sourcesJar(type: Jar) {
...
+    implementation "androidx.browser:browser:1.0.0"
```
Comment (iambmelt) [inline msal/build.gradle]: Same feedback here

File: msal/build.gradle [hunk 3]
```diff
@@ -116,9 +116,9 @@ dependencies {
     testImplementation "org.mockito:mockito-android:$rootProject.ext.mockitoAndroidVersion"
 
     // instrumentation test dependencies
-    androidTestImplementation "com.android.support.test:runner:$rootProject.ext.runnerVersion"
+    androidTestImplementation "androidx.test:runner:1.1.0"
```
Comment (iambmelt) [inline msal/build.gradle]: Same feedback here

File: msal/build.gradle [hunk 4]
```diff
@@ -116,9 +116,9 @@ dependencies {
...
     // Set this dependency to use JUnit 4 rules
-    androidTestImplementation "com.android.support.test:rules:$rootProject.ext.rulesVersion"
+    androidTestImplementation "androidx.test:rules:1.1.0"
```
Comment (iambmelt) [inline msal/build.gradle]: Same feedback here

File: msal/build.gradle (hunk 1 above)
Comment (rpdome) [reply msal/build.gradle]: I updated .gradle in every project (ad-accounts, common, msal, adal) and make them invoke the same versions.gradle file.

**Pull request merged**